pgxn/neon/pagestore_smgr.c

```diff
@@ -261,6 +261,26 @@ static inline void prefetch_set_unused(uint64 ring_index);
 static XLogRecPtr neon_get_request_lsn(bool *latest, NRelFileInfo rinfo,
 									   ForkNumber forknum, BlockNumber blkno);
 
+
+#define INITIAL_EXPONENTIAL_BACKOFF_DELAY 1000
```

Comment:
It's useful to name variables with a unit suffix when it's not obvious (I guess `_US` in these cases?)